docs/aws-copilot.md

```diff
@@ -46,72 +82,64 @@ sso_registration_scopes = sso:account:access
 
 Running `aws sso login` will log you in via your browser.
 
-After this, AWS Copilot commands should run on your new profile:
+NOTE: If you have multiple AWS accounts, you can also use temporary credentials and export them. These can be found in the Command line or programmatic access bit under your account.
 
-```bash
-$ copilot app ls
-manage-childrens-vaccinations
-```
+After this, AWS Copilot will have access and visibility of the account and you can start using it.
 
-## Manual deployment
+## AWS Copilot - Starting from scratch
 
-Assuming you have an environment setup, go ahead and deploy:
+If setting it up the first time, or after tearing down the existent stack, follow the instructions below:
 
-```bash
-copilot svc deploy --env staging
-```
+**If you need to tear it down**
 
-## Opening a shell on the remote environment
+`copilot app delete`, and follow the instructions
 
-If you have the service up and running, you can connect to the first running
-container with this command:
+**Setting up a new Stack**
 
-```bash
-copilot svc exec --app manage-childrens-vaccinations --env staging --name webapp
-```
-
-### Tailing logs of running service
-
-Use this command to see the most recent logs and to follow any new logs:
-
-```bash
-copilot svc logs --since 1h --follow
-```
-
-## Setting up a new environment
-
-Before you start, if you want HTTPS, you need to set up certificates in ACM.
+Before you start, if you want to use HTTPS, you need to set up certificates in ACM.
 
 The certificate needs to be verified by the DNS team by sending them the
 verification `cname`. On their end, they will verify the ownership, which in
 turn will update the status of the certificate to 'verified' in the ACM List of
 Certificates.
 
-Once the cert is approved, feed the ARN using the CLI:
+Once the cert is approved, continue the steps below.
 
-```bash
-copilot env init --import cert arn:aws.....
-```
+1. Do `aws configure sso`, or export the termporary credentials you got from the console start page.
+2. Initialise the app:
+   `copilot app init`, and follow the instructions
+3. Initialise the environment:
+   `$ copilot env init --name env_name`
+4. Deploy the enviornment:
+   `copilot env deploy --name staging`
+5. initialise the secrets if you need them:
+   ` copilot secret init --name secret_name`
+6. Initialise the service:
+   `copilot svc init` and follow the instructions. In our case it is a Load Balanced Web Service option
+7. Deploy the service to the target enviornment:
+   `copilot svc deploy --name service_name --env env_name`
```

Comment:
This formatting doesn't look great, the commands are nested inside the bullets:

<img width="1200" alt="Screenshot 2024-02-12 at 13 28 32" src="https://github.com/nhsuk/manage-childrens-vaccinations/assets/1650875/5e85f335-ab5d-4a27-a179-6da6bc3ae572">

I'd rather we do something like the cheat sheet where it's one bash section with comments for each line:

```bash
copilot app init # Initialise the app
copilot env init --name env_name # Initialise the environment
```

This makes it easier to copy and paste individual commands.